sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -1008,7 +1009,17 @@ class CompactBlockProcessor internal constructor(
          * granular information can be provided about scan state. Unfortunately, small batches may also lead to a lot
          * of overhead during scanning.
          */
-        internal const val SYNC_BATCH_SIZE = 100
+        private const val SYNC_BATCH_SIZE = 1000
+
+        /**
+         * This is the same as [SYNC_BATCH_SIZE] but meant to be used in the Zcash sandblasting periods
+         */
+        private const val SYNC_BATCH_SMALL_SIZE = 100
+
+        /**
+         * Known Zcash sandblasting period
+         */
+        private val SANDBLASTING_RANGE = 1_710_000L..2_050_000L
```

Comment:
I believe this to be roughly correct. At some point I'll delve back into the chain and figure out more precise boundaries.